examples/web_api.rs

```diff
@@ -1,89 +1,99 @@
-#![deny(warnings)]
```

Comment:
Could we keep the deny so we write examples without warnings?

Reply:
Didn't know that's what it was for. Thanks!